lib/fluent/plugin/output.rb

```diff
@@ -310,6 +310,10 @@ def configure(conf)
             log.warn "'flush_at_shutdown' is false, and buffer plugin '#{buf_type}' is not persistent buffer."
             log.warn "your configuration will lose buffered data at shutdown. please confirm your configuration again."
           end
+
+          if @flush_mode != :interval && buffer_conf.has_key?('flush_interval')
```

Comment:
> flush_interval is explicitly specified in configuration even if @chunk_key_time is true.

This means we should use `:interval` with following configuration?

```
<buffer>
  flush_interval 10s # use flush_mode interval
</buffer>
```

And warning for follwing configuration?

```
<buffer>
  flush_interval 10s
  flush_mode lazy
</buffer>
```